engine/src/cmd/collision.cpp

```diff
@@ -292,15 +292,24 @@ void Collision::collide( Unit* unit1,
     // See commit a66bdcfa1e00bf039183603913567d48e52c7a8e method Unit->jumpReactToCollision for an example
     // I assume this is for "always open" jump gates that you pass through
 
+    collision1.apply_force = false;
```

Comment:
This should not be necessary. The variables are initialized in the header.

Reply:
You might be right. But it wasn't obvious from looking at the code whether these fields were initialized to `false`, or not.

Reply:
You're right. I see that now. I just took those redundant initializations out.